app/feature/feature-changeaddress/src/main/kotlin/com/hedvig/android/feature/changeaddress/destination/ChangeAddressResultDestination.kt

```diff
@@ -1,31 +1,48 @@
-package com.feature.changeaddress.navigation
+package com.hedvig.android.feature.changeaddress.destination
 
-import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
 import androidx.compose.material3.Surface
-import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import com.hedvig.android.core.designsystem.component.button.LargeContainedButton
-import com.hedvig.android.core.ui.appbar.m3.TopAppBarWithBack
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.hedvig.android.core.designsystem.component.button.HedvigTextButton
+import com.hedvig.android.core.designsystem.component.success.HedvigSuccessSection
+import com.hedvig.android.core.designsystem.preview.HedvigPreview
+import com.hedvig.android.core.designsystem.theme.HedvigTheme
 
 @Composable
 internal fun ChangeAddressResultDestination(
-  navigateUp: () -> Unit,
+  movingDate: String?,
```

Comment:
Is it possible to have this be kotlinx.datetime.LocalDate?
Or does this come 100% straight from backend, so we don't want to deal with any conversions? If that is the case, then since I see we only use it to put it in a string, then we can keep it this way.